docs/source/couchdb_tutorial.rst

```diff
@@ -315,39 +313,39 @@ Therefore we need to install a chaincode on every peer that will execute and
 endorse our transactions. However, before we can interact with our chaincode,
 the members of the channel need to agree on a chaincode definition that
 establishes chaincode governance. In the previous section, we demonstrated how
-to add the index to the chaincode folder so that it is ready for deployment.
+to add the index to the chaincode folder so that the index is deployed with
+the chaincode .
 
 The chaincode needs to be packaged before it can be installed on our peers.
 We can use the `peer lifecycle chaincode package <http://hyperledger-fabric.readthedocs.io/en/latest/commands/peerlifecycle.html#peer-lifecycle-chaincode-package>`__ command
 to package the marbles chaincode.
 
 :guilabel:`Try it yourself`
 
-Assuming you have started the BYFN network, enter the CLI container:
-
-.. code:: bash
-
-    docker exec -it cli bash
-
-Your command prompt will change to something similar to:
+1. After you start the test network, copy and paste the following environment
+variables in your CLI to interact with the network as the Org1 admin. Make sure
+that you are in the `test-network` directory.
 
 .. code:: bash
 
-   bash-4.4#
+    export PATH=${PWD}/../bin:${PWD}:$PATH
+    export FABRIC_CFG_PATH=$PWD/../config/
+    export CORE_PEER_TLS_ENABLED=true
+    export CORE_PEER_LOCALMSPID="Org1MSP"
```

Comment:
nit: double quotes should be used around the `$PWD` references. I wouldn't bother changing it here but may be worth a separate task for some other time.

Reply:
done.